features/src/main/feature/camel-features.xml

```diff
@@ -1602,21 +1602,14 @@
         <bundle dependency='true'>wrap:mvn:ognl/ognl/3.4.2</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-ognl/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-olingo2' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <feature>http</feature>-->
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpcore-osgi/${httpcore4-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpclient-osgi/${httpclient4-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpasyncclient-osgi/${httpasyncclient-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:commons-codec/commons-codec/${commons-codec-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.commons/commons-lang3/${commons-lang3-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.google.code.gson/gson/${olingo2-gson-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.olingo/olingo-odata2-api/${olingo2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.olingo-odata2-core/${olingo-odata2-core-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.jsr311-api-1.1.1/${servicemix-specs-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.camel/camel-olingo2-api/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-olingo2/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-olingo2' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version="[4,5)">http-client</feature>
+        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpasyncclient-osgi/4.1.5</bundle>
+        <bundle dependency='true'>mvn:org.apache.commons/commons-lang3/3.14.0</bundle>
```

Comment:
```suggestion
        <bundle dependency='true'>mvn:org.apache.commons/commons-lang3/${commons-lang3-version}</bundle>
```
I know that we hard code for now the versions to avoid conflicts but let's reuse the existing maven properties